### DISCUSSION TITLE
Just to another session's panel in one shot

### DISCUSSION THREAD

**Discussion by LintaoAmons** [jrmoulton/tmux-sessionizer#135] [Ideas]
So I could have multiple sessions and multiple windows and multiple panels.
If I need to navigate to a specific panel, I first need to jump to that session, find the window, and then go to the panel.
Is it possible to have one of window, just like how I select the session, but with all the existing panels so I just jump to that panel in a specific window of the specific session with one shortcut.

**Comment by jrmoulton** [comment]
This isn't currently possible but it could be added
